packages/analytics-js-integrations/__tests__/integrations/GA4/browser.test.js

```diff
@@ -80,15 +71,20 @@ describe('Google Analytics 4 Page events tests', () => {
         expect(error.message).toEqual(output.message);
       }
     });
-  })
+  });
 });
 
 describe('Google Analytics 4 Group events tests', () => {
   let ga4;
   beforeEach(() => {
     ga4 = new GA4(
-      { measurementId: 'G-123456', extendPageViewParams: true, capturePageView: 'rs', piiPropertiesToIgnore: [{ piiProperty: '' }] },
-      { getUserId: () => '1234', getUserTraits: () => { } },
+      {
```

Comment:
input.config can be used right?

Reply:
we are taking configs from test case because for each test config is changing in page call, for for rest of the call config is same so no need to add it in tests payload